tests/connector-service-tests/test/testkit/connectorclient.go

```diff
@@ -66,6 +67,25 @@ func (cc connectorClient) CreateToken(t *testing.T) TokenResponse {
 	return tokenResponse
 }
 
+func (cc connectorClient) RevokeCertificate(t *testing.T, revocationUrl, csr string) *http.Response {
+	revocationBody := RevocationBody{
+		Hash: csr,
```

Comment:
As discussed we should pass here hash of the certificate instead of CSR.

Reply:
Done!